lib/core/text/abstract_text.nit

```diff
@@ -602,7 +605,13 @@ abstract class Text
 	#     assert "{0.code_point}{1.code_point}{8.code_point}{31.code_point}{32.code_point}".escape_to_c == "\\000\\001\\010\\037 "
 	#
 	# The exceptions are the common `\t` and `\n`.
-	fun escape_to_c: String
+	fun escape_to_c: String do return escape_to_c_text_impl
+
+	# The original definition of `escape_to_c`
+	#
+	# This method is separated from `escape_to_c` mainly for testing purposes.
+	# Normally, you should not call this method explicitly nor redefine it.
+	protected fun escape_to_c_text_impl: String
```

Comment:
I really don't see why, if you want to test the method on receivers of each type, you can already do it by adding a test, instanciate each Text subclass, and check the result is really what you expected, there's no need to complicate an already complex hierarchy by either adding a proxy test method or splitting the modules.